packages/nocodb/src/models/AIColumn.ts

```
@@ -0,0 +1,48 @@
+    return data && new AIColumn(data);
+  }
+
+  public static async insert(
+    context: NcContext,
+    aiColumn: Partial<AIColumn> & {
+      fk_model_id: string;
+      fk_column_id: string;
+    },
+    ncMeta = Noco.ncMeta,
+  ) {
+    return this._insert(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)